spec/slappy/slack_api/concerns/findable_spec.rb

```diff
@@ -3,17 +3,35 @@
 describe Slappy::SlackAPI::Findable do
   class Group
     include Slappy::SlackAPI::Findable
+    self.monitor_event = 'group_open'
   end
 
-  before { allow(Slack).to receive(:groups_list).and_return(data) }
+  before { allow(Slack).to receive(:groups_list).and_return(group_list) }
   let(:test_class) { Group }
-  let(:data) { { 'groups' => [{ id: id, name: name }] } }
+  let(:subtype)    { 'group_open' }
+  let(:group_list) { { 'groups' => [data] } }
+  let(:data) { { id: id, name: name, subtype: subtype, ts: Time.now } }
```

Comment:
Rails/TimeZone: Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.